pygments/lexers/wren.py

```diff
@@ -0,0 +1,97 @@
+"""
+    pygments.lexers.wren
+    ~~~~~~~~~~~~~~~~~~~~
+    Lexer for Wren.
+    :copyright: Copyright 2006-2022 by the Pygments team, see AUTHORS.
+    :license: BSD, see LICENSE for details.
+"""
+
+import re
+
+from pygments.lexer import RegexLexer
+from pygments.token import Whitespace, Punctuation, Keyword, Name, Comment, \
+    Operator, Number, String
+
+__all__ = ['WrenLexer']
+
+class WrenLexer(RegexLexer):
+    name = 'Wren'
+    aliases = ['wren']
+    filenames = ['*.wren']
+
+    flags = re.MULTILINE | re.DOTALL
+
+    tokens = {
+        'root': [
+            # Whitespace.
+            (r'\s+', Whitespace),
+            (r'[,\\\[\]{}]', Punctuation),
+
+            # Push a parenthesized state so that we know the corresponding ')'
+            # is for a parenthesized expression and not interpolation.
+            (r'\(', Punctuation, ('parenthesized', 'root')),
+
+            # In this state, we don't know whether a closing ')' is for a
+            # parenthesized expression or the end of an interpolation. So, do
+            # a non-consuming match and let the parent state (either
+            # 'parenthesized' or 'interpolation') decide.
+            (r'(?=\))', Whitespace, '#pop'),
+
+            # Keywords.
+            (r'(as|break|class|construct|continue|else|for|foreign|if|import|'
```

Comment:
Please use the `words()` function to create a match for all those names.

Reply:
Fixed. Note also that I've decided to reclassify 'in' and 'is' as 'Operator.Word' rather than 'Keyword'.